src/main/java/it/gov/pagopa/pu/bff/service/debt_position/DebtPositionRetrieverService.java

```diff
@@ -13,4 +13,7 @@ public interface DebtPositionRetrieverService {
   PagedDebtPositionView getDebtPositionViews(DebtPositionViewFiltersDTO filtersDTO, Pageable pageable, UserInfo loggedUser, String accessToken);
 
   DebtPositionDetailDTO getDebtPositionDetail(Long debtPositionId, Long organizationId, UserInfo loggedUser, String accessToken);
+
+  Boolean deleteDebtPosition(Long organizationId, Long debtPositionId, UserInfo loggedUser, String accessToken);
```

Comment:
let's add a javadoc to describe what's the meaning of true or false
```suggestion
  boolean deleteDebtPosition(Long organizationId, Long debtPositionId, UserInfo loggedUser, String accessToken);
```